controllers/utils_test.go

```diff
@@ -792,8 +792,11 @@ func makeTestReconciler(obj ...client.Object) *RolloutManagerReconciler {
 	cl := fake.NewClientBuilder().WithScheme(s).WithStatusSubresource(obj...).WithObjects(obj...).Build()
 
 	return &RolloutManagerReconciler{
-		Client:                       cl,
-		Scheme:                       s,
+		Client: cl,
+		Scheme: s,
+		ResourceLabels: map[string]string{
+			"custom1": "value",
+		},
```

Comment:
Rather than adding a custom label to the object generated for ALL unit tests that call `makeTestReconciler`,  it would be better to only add ResourceLabels field  for unit tests that verify the custom labels feature